xml/System.Net.Security/SslStreamCertificateContext.xml

```diff
@@ -38,12 +38,15 @@
         <Parameter Name="offline" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="target">To be added.</param>
-        <param name="additionalCertificates">To be added.</param>
-        <param name="offline">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="target">The server certificate.</param>
+        <param name="additionalCertificates">The certificates to chain.</param>
+        <param name="offline">
+          <see langword="true" /> to indicate that the missing certificates can be downloaded; otherwise, <see langword="false" />.</param>
```

Comment:
otherwise only available X509Certificate stores will be searched for missing certificates.